borrowing/apps.py

```diff
@@ -1,6 +1,15 @@
 from django.apps import AppConfig
+from django.db.models.signals import post_migrate
+
+
+def post_migrate_schedule_tasks(sender, **kwargs):
+    from .scheduled_tasks import schedule_tasks
```

Comment:
Maybe write an import at the top of the file

Reply:
This is intended: those imports will break the code if imported globally. They have to be imported locally, inside a function.